src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -116,7 +117,10 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .Setup<string>("dotnet-framework-version")
                 .WithDescription("Only applies to dotnet-isolated applications. Specifies the .NET Framework version for the function app. For example, set to '5.0' when publishing a .NET 5 app.")
                 .Callback(s => DotnetFrameworkVersion = s);
-
+            Parser
+                .Setup<bool>("deploy-only")
```

Comment:
Avoid adding `deploy-only` flag  - i do not see any other reasons why will users explicitly pass this flag . If this is intended experience for KubeApp - we should do that by default and let the users pass in verbose flag to see rest of the logs

Reply:
Let me clarify your idea and understand the context. I'd happy to remove the `deploy-only` option.  I'd like to explain the context when I tested it. 
func azure functionapp publish command  is deploy app and shows the list of the triggers.  The behavior is fine, however, we need to wait around 30 sec to see the updated value.  Usually it's useful, however, some people don't need to see the information is updated and don't want to wait to finish the command.

```
Functions in tsushiqueue1907:
    QueueTrigger12 - [queueTrigger]
```

So that I just want to give the customer to have an option whether if they need this info with 30 sec or not. (at least for Kubernetes Environment, I don't know when I do it for other platform) 
In this case, in you idea, in case, by default, in kubernetes environment, we don't need the trigger info, and `--verbose` option reside in the `StartHostAction` In this case, should we move the `VersboseLogging` from `StartHostAction` to `BaseAction` for references from `PublishFunctionAppAction`?  I also think `--verbose` option looks little different for showing trigger information.  The other idea is removing the showing info feature for Kubernetes Environment. 
Do you want to avoid to increase a new behavior for avoiding to introduce new bugs?

Reply:
Thanks for explanation. My recommendation is do not add a separate flag - longer delay to display Functions Info for KubeApps is ok - as long as the long polling is scoped to KubeApp.

Also, displaying logs while polling for functions host status is OK!

Reply:
Sure. Then I'll simply remove the `--deploy-only` flag. Thank you for clarifying it!

Reply:
Hi @pragnagopa I removed the debug only flag and test it.